cirq-core/cirq/ops/global_phase_op.py

```diff
@@ -12,18 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """A no-qubit global phase operation."""
-from typing import Any, Dict, Sequence, Tuple, TYPE_CHECKING
+from typing import AbstractSet, Any, Dict, Sequence, Tuple, TYPE_CHECKING, Union
 
 import numpy as np
+import sympy
 
 from cirq import value, protocols
 from cirq._compat import deprecated_class
+from cirq.type_workarounds import NotImplementedType
 from cirq.ops import gate_operation, raw_types
 
 if TYPE_CHECKING:
     import cirq
 
 
+ComplexParam = Union[value.Scalar, sympy.Basic]
```

Comment:
nit: 

```suggestion
TParamComplex = Union[value.Scalar, sympy.Basic]
```

Also, should we define it in cirq-core/cirq/value/type_alias.py so we don't need to redefine in pauli_string.py below?